cmd/agent/app/run.go

```diff
@@ -253,6 +253,16 @@ func StartAgent() error {
 	}
 
 	// Setup Internal Profiling
+	if v := config.Datadog.GetInt("internal_profiling.block_profile_rate"); v > 0 {
```

Comment:
I am not sure we need a configuration option from `datadog.yaml`. Do you have use case in mind?

Reply:
If we enable internal profiling in the config, we may want to configure these optional profile types at the same time.